test/constructors.jl

```diff
@@ -302,11 +292,36 @@ end
     @test map(typeof, eachcol(df)) == answer
 end
 
-@testset "categorical constructor" begin
-    df = DataFrame([Int, String], [:a, :b], [false, true], 3)
-    @test !(df[:a] isa CategoricalVector)
-    @test df[:b] isa CategoricalVector
-    @test_throws DimensionMismatch DataFrame([Int, String], [:a, :b], [true], 3)
+@testset "constructor with types" begin
+    df = DataFrame([Union{Int, Missing}, Union{Float64, Missing}, Union{String, Missing}],
+                   [:A, :B, :C], 100)
+    @test size(df, 1) == 100
+    @test size(df, 2) == 3
+    @test typeof(df[1]) == Vector{Union{Int, Missing}}
+    @test typeof(df[2]) == Vector{Union{Float64, Missing}}
+    @test typeof(df[3]) == Vector{Union{String, Missing}}
+    @test all(ismissing, df[1])
+    @test all(ismissing, df[2])
+    @test all(ismissing, df[3])
+    @test typeof(df[:, 1]) == Vector{Union{Int, Missing}}
```

Comment:
Is this really useful? `df[1]` should be enough, right (I know it was in the original code)?

Reply:
removed (it was kind of preparation for `df[1]` deprecation, but not I am not sure if we will deprecate it after all)